src/tests/device_add_remove.c

```diff
@@ -21,7 +21,7 @@
 #include <lxc/lxccontainer.h>
 
 #define NAME "device_add_remove_test"
-#define DEVICE "/dev/network_latency"
+#define DEVICE "/dev/null"
```

Comment:
That doesn't work, we need to add/remove a device which isn't pre-created by liblxc.

Reply:
hmkay

Reply:
switched to /dev/loop-control